### DISCUSSION TITLE
IDE = PyCharm, evernote-backup installed via pip - how to execute?

### DISCUSSION THREAD

**Discussion by SprickW** [vzhd1701/evernote-backup#22] [General]
Commands like "evernote-backup ..." can not be found.

docker currently not installed? Should not be needed, right?

Greetings Wolfgang

**Comment by RhetTbull** [comment]
My guess is you are in a different python virtual environment than where you installed it. If you simply want to run `evernote-backup` I strongly suggest you install with brew or pipx as these will place the executable on your path. If you installed with pip, you should be able to do `python3 -m evernote-backup` but it all depends on how your virtual environments are set up. Are you trying to run this inside the PyCharm terminal or in the Terminal.app?  Where did you run pip (inside PyCharm or in Terminal.app)?

**Reply by SprickW** [reply]
Using pip in PyCharm venv still causes 
   /home/zzz/PycharmProjects/evernote-backup/venv/bin/python3: No module named evernote-backup

Using pipx from wirth PyCharm venv also causes problems.

So I followed your advice to use pipx - and as far as I am - everything seems fine :-)

**Reply by vzhd1701** [reply]
You should use `evernote_backup` if you are trying to execute it as a module `python3 -m evernote_backup`.
